src/main/java/com/revature/controllers/QuestionEvalController.java

```diff
@@ -0,0 +1,62 @@
+package com.revature.controllers;
+
+import com.revature.domain.QuestionComment;
+import com.revature.domain.QuestionPool;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.revature.domain.QuestionPool'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AUselessImportCheck)